tools/submission/submission-checker.py

```diff
@@ -286,7 +286,7 @@ def split_path(m):
     return m.replace("\\", "/").split("/")
 
 
-def ignore_errors_for_v0_5(line):
+def ignore_errors(line):
     if "check for ERROR in detailed" in line:
         return True
     if "Loadgen built with uncommitted changes" in line:
```

Comment:
Should we still allow this?

Reply:
I think we talked about ignoring 'CAS' errors. Not sure about others.

Reply:
ok, I hang now the errors on the mlperf version and only allow cas for v0.7.